lib/Data/TextOrBinary.rakumod

```diff
@@ -0,0 +1,36 @@
+my constant PRINTABLE_TABLE = do {
+    my int @table;
+    @table[flat ords("\t\b\o33\o14"), 32..126, 128..255] = 1 xx *;
+    @table
+}
+
+my uint $ten      = 10;
+my uint $thirteen = 13;
+
```

Comment:
Why? If this makes a difference it's Rakudo that should be fixed, given the other side of the comparison is clearly `uint`.

Reply:
````
% r 'my uint $a = 10; my $b := $a == $a for ^10000000' 
real	0.42s
user	0.43s
sys	0.02s
% r 'my uint $a = 10; my $b := $a == 10 for ^10000000'
real	1.30s
user	1.31s
sys	0.02s
````
Which is caused by this dispatch problem:
````
$ r 'multi sub a(uint $, uint $) { dd }; multi sub a(Int $, Int $) { dd }; my uint $a = 10; a $a, 10; a $a, $a'
sub a(Int, Int)
sub a(uint, uint)
````
So clearly `10` is *not* a `uint`.

BTW, the same goes for `int`:
````
% r 'multi sub a(int $, int $) { dd }; multi sub a(Int $, Int $) { dd }; my int $a = 10; a $a, 10; a $a, $a' 
sub a(Int, Int)
sub a(int, int)
````
So clearly, `10` is not an `int` either.

See also discussion yesterday: https://irclogs.raku.org/moarvm/2022-08-01.html#18:47-0002